woocommerce/woocommerce-ios
# Fix of the WCShip extension name in the app

Part of #6914

### Description
While testing the Shipping Label onboarding banner for the installation of the WCShip extension in the Order Detail, I noticed that for one of my sites the banner should not be visible since the extension is installed and I can also create a shipping label. It is similar to the problem reported by @Ecarrion [yesterday](https://a8c.slack.com/archives/C6H8C3G23/p1654531437701509), where I thought that the PR for eligibility had not yet been merged.
Looking better at the issue, it seems that we were using the wrong name for the plugin extension, which is called `WooCommerce Shipping &amp; Tax`, so basically the name returned is the HTML one 😿 

### Testing instructions
1. Launch the app, and select a website without the WCShip extension installed (you can create a Jurassic ninja website, like a did). The feature flag `shippingLabelsOnboardingM1` should be enabled (enabled by default in debug).
2. Open the Order List.
3. Select an order eligible for Shipping Labels.
4. You should be able to see the banner after the products list.
5. By tapping on the banner, nothing should happen. 6. 

Do the same operation, on a website with WCShip installed. You should not be able to see the banner in the order of detail.

---
